### PR TITLE
Bump weekly and CI updates

### PR DESCRIPTION
**Pull request by ThetaSinner**

### CODE REVIEW

File: .github/workflows/performance.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ env:
   INFLUX_TOKEN: ${{ secrets.INFLUX_TOKEN }}
   WT_METRICS_DIR: "${{ github.workspace }}/telegraf/metrics"
 
+concurrency:
```
Comment (ThetaSinner) [inline .github/workflows/performance.yaml:10]: I think we want to prevent this running more than once at the same time?

File: flake.nix [hunk 2]
```diff
@@ -100,8 +100,6 @@
         };
 
         devShells.ci = pkgs.mkShell {
-          inputsFrom = [ inputs'.holonix.devShells ];
```
Comment (ThetaSinner) [inline flake.nix:103]: Wrong syntax, doesn't do anything

**Review by cdunster — APPROVED** [review]

**Review by c12i — APPROVED** [review]

File: .github/workflows/performance.yaml (hunk 1 above)
Comment (ThetaSinner) [reply .github/workflows/performance.yaml:10]: It's more about not being able to trust the results if mutliple runs are attempted concurrently on the machine

**Review by neonphog — APPROVED** [review]

**Comment by ThetaSinner** [comment]
Closing in favour of #101 which has the same changes plus some more and a newer Holochain version

**Pull request closed without merging**